homeassistant/components/envisalink/switch.py

```diff
@@ -0,0 +1,74 @@
+"""Support for Envisalink zone bypass switches."""
+import logging
+
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from . import (
+    CONF_ZONENAME,
+    DATA_EVL,
+    SIGNAL_ZONE_BYPASS_UPDATE,
+    ZONE_SCHEMA,
+    EnvisalinkDevice,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the Envisalink switch devices."""
+    configured_zones = discovery_info["zones"]
+
+    devices = []
+    for zone_num in configured_zones:
+        device_config_data = ZONE_SCHEMA(configured_zones[zone_num])
+        device = EnvisalinkSwitch(
+            hass,
+            zone_num,
+            device_config_data[CONF_ZONENAME] + "_bypass",
+            hass.data[DATA_EVL].alarm_state["zone"][zone_num],
+            hass.data[DATA_EVL],
+        )
+        devices.append(device)
+
+    async_add_entities(devices)
```

Comment:
```suggestion
    entities = []
    for zone_num in configured_zones:
        device_config_data = ZONE_SCHEMA(configured_zones[zone_num])
        entities = EnvisalinkSwitch(
            hass,
            zone_num,
            device_config_data[CONF_ZONENAME] + "_bypass",
            hass.data[DATA_EVL].alarm_state["zone"][zone_num],
            hass.data[DATA_EVL],
        )
        entities.append(device)

    async_add_entities(entities)
```

These are `entities` not `devices`

Reply:
I've updated this as well as the other naming inconsistencies for the other entities.